i18n/zh_CN.po

```diff
@@ -35,83 +43,103 @@ msgstr "所有库服务器"
 msgid "Translations"
 msgstr "语言"
 
+# index.html
+msgid "Introduction"
+msgstr "导言"
+
+msgid "Active Servers"
+msgstr "活动服务器"
+
+msgid "Links"
+msgstr "链接"
+
+msgid "Terms of service"
+msgstr "服务条款"
+
+msgid "Subscribe in a reader"
+msgstr "用阅读器订阅"
+
+msgid "Older news"
+msgstr "旧新闻"
+
 # mailinglists.html
-msgid  "NTP Pool mailing lists"
-msgstr "NTP池-邮件清单"
+msgid "NTP Pool mailing lists"
+msgstr "NTP 池邮件列表"
 
-msgid  "subscribe"
+msgid "subscribe"
 msgstr "订阅"
 
-msgid  "archive"
+msgid "archive"
 msgstr "文章归档"
 
-msgid  "Announcement list"
+msgid "Announcement list"
 msgstr "公告列表"
 
-msgid  "Discussion list"
+msgid "Discussion list"
 msgstr "讨论列表"
 
-msgid  "Development list"
+msgid "Development list"
 msgstr "开发列表"
 
-msgid  "announcement_list_description"
-msgstr "公告列表简介"
+# This usage is so un-gettext-ish... Oh my.
+msgid "announcement_list_description"
+msgstr "NTP Pool 的公告邮件量非常的少，请所有服务器管理员务必要订阅！"
 
-msgid  "discussion_list_description"
-msgstr "讨论列表简介"
+msgid "discussion_list_description"
+msgstr "不管你是服务器管理员还是用户，如果你想更多的参与这个项目，请订阅我们的讨论清单。<br/> 一般有关 NTP 的讨论则应该在 %1 Usenet 新闻群组上（或是在 <a href=\"%2\">Google Groups</a>）"
 
-msgid  "development_list_description"
-msgstr "开发列表简介"
+msgid "development_list_description"
+msgstr "更多有关目前开发 <a href=\"%1\">ntp pool 代码</a> 以及 <a href=\"%2\">pgeodns</a> 相关技术的讨论会在 timekeepers-dev 清单上。"
 
-msgid  "irc_channel_description"
-msgstr "IRC频道简介"
+msgid "irc_channel_description"
+msgstr "我们在 %1 服务器提供一个叫做 %2 的 IRC 频道给所有用户以及维护人员。"
 
 # tpl/server.html
-msgid  "Back to the front page"
+msgid "Back to the front page"
 msgstr "回到首页"
 
-msgid  "Find"
+msgid "Find"
 msgstr "搜索"
 
-msgid  "Stats for %1"
+msgid "Stats for %1"
 msgstr "%1的统计信息"
 
-msgid  "Not active in the pool, monitoring only"
+msgid "Not active in the pool, monitoring only"
 msgstr "在池中不可用，只能进行查看"
 
-msgid  "Zones:"
-msgstr "区域:"
+msgid "Zones:"
+msgstr "区域："
 
-msgid  "This server is <span class=\"deletion\">scheduled for deletion</span> on %1."
-msgstr "这个服务器预计于 %1 被删除"
+msgid "This server is <span class=\"deletion\">scheduled for deletion</span> on %1."
+msgstr "这个服务器预计于 %1 被<span class=\"deletion\">删除</span>。"
 
-msgid  "Current score: %1 (only servers with a score higher than %2 are used in the pool)"
+msgid "Current score: %1 (only servers with a score higher than %2 are used in the pool)"
 msgstr "目前得分: %1 (只有得分高于 %2 的服务器会被存入池中)"
 
-msgid  "What do the graphs mean?"
+msgid "What do the graphs mean?"
 msgstr "这些图表意味着什么？"
 
-msgid  "CSV log"
+msgid "CSV log"
 msgstr "CSV 日志"
 
 # tpl/navigation_sidebar.html
-msgid  "News"
+msgid "News"
 msgstr "新闻"
 
-msgid  "How do I <i>use</i> pool.ntp.org?"
-msgstr "我如何能<i>使用</i> pool.ntp.org?"
+msgid "How do I <i>use</i> pool.ntp.org?"
+msgstr "如何<u>使用</u> pool.ntp.org？"
```

Comment:
Chinese is generally unsuitable for italics, therefore using underline for emphasis instead.